### ISSUE TITLE
Chrome warnings about preventDefault of passive targets

### ISSUE THREAD

**Issue by jmtoball** [Bernardo-Castilho/dragdroptouch#6]
Hello and thank you for this polyfill. It's exactly what we've been looking for. There's just one small issue:
Chrome (probably starting at version 56) seems to not be happy about the preventDefault-calls as it treats the touch-listeners as passive. This does not seem to interfere with the functionality, but for a simple drag-process, ~100 of these warnings are logged to the console:
`[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080`
I fear that this may also be discouraged for mobile browsers and may become unsupported in the future, so I thought I'd give you a heads-up.

**Comment by Bernardo-Castilho** [comment]
Thank you for the feedback. I had seen those messages and they bothered me as well, but it wasn't obvious to me how I could detect the passive listener support. When I saw your message, I searched a bit and found a way to do it (https://github.com/Modernizr/Modernizr/issues/1894).

The latest version adds the listeners as non-passive, and that seems to solve the issue (no messages on the console).

Thanks a lot for pointing this out, 

PS: Some articles suggested using CSS to prevent the standard touch behaviors instead of using non-passive event listeners. I tried that approach as well but couldn't make it work properly.
